app/jobs/github_commits_job.rb

```diff
@@ -8,6 +8,7 @@ def run
       begin
         update_total_commit_count_for(project)
         update_user_commit_counts_for(project)
+        update_last_pushed_dt_for(project)
```

Comment:
so perhaps we want to have these in separate rescues so one failure doesn't block the others?